porteux-scripts/porteux-app-store/porteux-app-store.py

```diff
@@ -8,12 +8,23 @@
 import subprocess
 from subprocess import run, Popen
 import signal
+import json
 
 if os.geteuid() != 0:
     this_script = path.abspath(__file__)
     run(['psu', this_script])
     quit()
 
+GTK_DIALOG_SCRIPT = "/opt/porteux-scripts/gtkdialog.py"
+GTK_PROGRESS_SCRIPT = "/opt/porteux-scripts/gtkprogress.py"
+APP_STORE_PATH = "/mnt/sdd3/Fun/porteux/porteux-scripts/porteux-app-store/"
```

Comment:
I believe we should change this to `/opt/porteux-scripts/porteux-app-store`

Reply:
You're right, I forgot to change, thanks

Reply:
:+1: